core/src/builtin_widgets/box_decoration.rs

```diff
@@ -136,20 +124,16 @@ impl BoxDecoration {
       Size::new(0., -border.bottom.width / 2.),
       Size::new(border.left.width / 2., 0.),
     ];
-    edges
-      .iter()
-      .zip(borders.iter())
-      .zip(borders_offset.iter())
-      .for_each(|((edge, border), offset)| {
+    edges.iter().zip(borders.iter()).zip(borders_offset.iter()).for_each(
```

Comment:
This chain writing style is a poor reading experience

Reply:
Are you suggesting that the "chaining style" or `one-line chaining` results in a poor reading experience?